Sources/DeepLearning/Loss.swift

```diff
@@ -16,14 +16,38 @@
 import TensorFlow
 #endif
 
+/// Computes the mean squared error between logits and labels.
+///
+/// - Parameters
+///   - logits: one-hot encoded outputs from a neural network.
+///   - labels: one-hot encoded values that correspond to the correct output.
 @differentiable
 public func meanSquaredError<Scalar: TensorFlowFloatingPoint>(
     predicted: Tensor<Scalar>, expected: Tensor<Scalar>) -> Tensor<Scalar> {
     return (expected - predicted).squared().mean()
 }
 
+/// Computes the softmax cross entropy (categorical cross entropy) between logits and labels.
+///
+/// - Parameters
+///   - logits: one-hot encoded outputs from a neural network.
+///   - labels: one-hot encoded values that correspond to the correct output.
 @differentiable
 public func softmaxCrossEntropy<Scalar: TensorFlowFloatingPoint>(
     logits: Tensor<Scalar>, labels: Tensor<Scalar>) -> Tensor<Scalar> {
     return -(labels * logSoftmax(logits)).mean(alongAxes: 0).sum()
 }
+
+/// Computes the sigmoid cross entropy (binary cross entropy) between logits and labels.
+///
+/// - Parameters
+///   - logits: single continuous values from 0 to 1.
+///   - labels: integer values that correspond to the correct output.
+@differentiable
+public func sigmoidCrossEntropy<Scalar: TensorFlowFloatingPoint>(
+    logits: Tensor<Scalar>, labels: Tensor<Scalar>) -> Tensor<Scalar> {
```

Comment:
Move `) -> Tensor<Scalar> {` to the next line.

See line wrapping guidelines here: https://google.github.io/swift/#line-wrapping.

Reply:
Done - thanks!